index.html

```diff
@@ -0,0 +1,278 @@
+            </li>
+          </ul>
+        </header>
+        <!-- header  section -->
+        <section class="first-section">
+          <div class="header-intro">
+            <h1 class="header-text">
+              Hey There.
+              I’m Promise
+            </h1>
+            <h2 class="header-smaller-text"> I am a Software Developer </h2>
+            <p class="header-paragraph">
+              I can help you build a product,
+              feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don’t hestiate to contact me.
+            </p>
+          </div>
+          <!-- Icons section -->
+          <div class="header-icons">
+            <ul>
+              <li><a href="https://https://promzzykoncepts.github.io/Template-2-Mobile-Version//"><img src="images/github.png"
+                    alt="Github logo"></a></li>
+              <li><a href="https://www.linkedin.com/in/promiseokechukwu/"><img src="images/Linkedin-icon.svg"
+                    alt="Linkedin logo"></a></li>
+              <li><a href="https://www.github.com/promzzykoncepts"><img src="images/angel-list.svg" alt="Angelist logo"></a></li>
+              <li><a href="https://medium.com/promiseoechukwu/"><img src="images/medium.png" alt="Medium logo"></a></li>
+              <li><a href="https://twitter.com/promzzy/"><img src="images/twitter.svg" alt="Twitter logo"></a>
+              </li>
+            </ul>
+          </div>
+        </section>
+      </div>
+      <!-- The Work section -->
+      <section id="Portfolio" class="work-section">
+        <header class="work-section-heading">
+          <h2>My Recent Works</h2>
+          <div class="line"></div>
+          <!-- <hr class="hr"> -->
+        </header>
+        <div class="work-section">
+          <div class="workSection1">
+            <div class="work-section-image">
+              <img class="img" src="./images/image1.png" alt="product image of an athlete">
+            </div>
+            <section class="work-section-texts">
+              <h3 class="work-section-text1">Multi-Post Stories</h3>
+              <p class="paragraph">
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's
+                standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+              </p>
+              <div class="work-section-ul">
+                <ul>
+                  <li class="btn">css</li>
+                  <li class="btn">html</li>
+                  <li class="btn">bootstrap</li>
+                  <li class="btn">Ruby</li>
+                </ul>
+              </div>
+              <button type="button" id="project-button1" class="project-btn1 project-btn">See Project</button>
+            </section>
```

Comment:
- [ ] Kindly the font size , the design and the text-color are not match to the figma template as requested in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
Your design
![Screenshot from 2022-06-30 22-54-31](https://user-images.githubusercontent.com/77842237/176766780-f5a669bc-55fa-4cda-9348-4aab8b6dab64.png)
Figma design
![Screenshot from 2022-06-30 22-53-57](https://user-images.githubusercontent.com/77842237/176766909-7a6200b6-cefa-4b23-9c89-1630c1db1785.png)

Reply:
@vic778 
Hello,
Thanks for the review
Although I think you got this one mixed up
I am using template 2 but you seem to be comparing it with template 1
Kindly recheck template 2

If am wrong, please respond.

Reply:
@PromzzyKoncepts Kindly my portfolio also is template 2. the color of the see-button first is not match to the Figma template as you can see at this screenshot . secondly the font-family you're using is not match. you may used the `font-family: Inter, sans-serif;` to make your text looks like the figma